### PR TITLE
Upgrade to Kotlin 1.3.71, ktor 1.3.2, serialization 0.20.0

### PR DESCRIPTION
**Pull request by spinach**
Solves #160

### CODE REVIEW

File: src/commonTest/kotlin/suite/TestSuiteMultiCluster.kt [hunk 1]
```diff
@@ -29,8 +29,8 @@ internal class TestSuiteMultiCluster {
         return "$prefix-$username-$id".toUserID()
     }
 
-    private suspend fun CoroutineScope.waitForUserID(userID: UserID) {
-        while (isActive) {
+    private suspend fun waitForUserID(userID: UserID) {
```
Comment (q-litzler) [inline src/commonTest/kotlin/suite/TestSuiteMultiCluster.kt:32]: I didn't know about this convention, do you have a reference I could read somewhere ? :)

Comment (Aallam) [reply src/commonTest/kotlin/suite/TestSuiteMultiCluster.kt:32]: https://medium.com/@elizarov/coroutine-context-and-scope-c8b255d59055

Comment (q-litzler) [reply src/commonTest/kotlin/suite/TestSuiteMultiCluster.kt:32]: Thanks !

**Comment by b95505017** [comment]
> Solves #160

I would like be beta tester.

**Pull request merged**